.eslintrc.json

```diff
@@ -1,41 +1,41 @@
 {
-  "env": {
-    "browser": true,
-    "node": true,
-    "es6": true,
-    "jest": true
-  },
-  "extends": [
-    "eslint:recommended",
-    "airbnb",
-    "plugin:react/recommended",
-    "plugin:import/errors",
-    "plugin:import/warnings"
-  ],
-  "parserOptions": {
-    "ecmaFeatures": {
-      "jsx": true
+    "env": {
+      "browser": true,
+      "node": true,
+      "es6": true,
+      "jest": true
     },
-    "ecmaVersion": 2018,
-    "sourceType": "module"
-  },
-  "rules": {
-    "no-debugger": "off",
-    "no-console": "off",
-    "no-unused-vars": "warn",
-    "react/prop-types": "warn",
-    "react/prefer-stateless-function": 0,
-    "react/jsx-filename-extension": [
-      1,
-      {
-        "extensions": [".js", ".jsx"]
+    "extends": [
+      "eslint:recommended",
+      "airbnb",
+      "plugin:react/recommended",
+      "plugin:import/errors",
+      "plugin:import/warnings"
+    ],
+    "parserOptions": {
+      "ecmaFeatures": {
+        "jsx": true
+      },
+      "ecmaVersion": 2018,
+      "sourceType": "module"
+    },
+    "rules": {
+      "no-debugger": "off",
+      "no-console": "off",
+      "no-unused-vars": "warn",
+      "react/prop-types": "warn",
+      "react/prefer-stateless-function": 0,
+      "react/jsx-filename-extension": [
+        1,
+        {
+          "extensions": [".js", ".jsx"]
+        }
+      ]
+    },
+    "settings": {
+      "react": {
+        "version": "detect"
       }
-    ]
-  },
-  "settings": {
-    "react": {
-      "version": "detect"
-    }
-  },
-  "root": true
-}
```

Comment:
Was there a need to make changes in this file.

Reply:
> Was there a need to make changes in this file.

There was no need and i have not made any changes to this file.